docs/mindsdb_sql/agents/agent.mdx

```diff
@@ -3,145 +3,173 @@ title: Agent
 sidebarTitle: Agent
 ---
 
-With MindsDB, you can create and deploy AI agents that comprise AI models and customizable skills such as knowledge bases and text-to-SQL.
+With MindsDB, you can create and deploy AI agents that comprise AI models and customizable skills such as [knowledge bases](/mindsdb_sql/knowledge-bases) and text-to-SQL.
 
 <p align="center">
   <img src="/assets/agent_diagram.png" />
 </p>
 
 AI agents use a conversational model (like OpenAI) from LangChain utilizing [tools as skills](https://python.langchain.com/docs/modules/agents/tools/) to respond to user input. Users can customize AI agents with their own prompts to fit their use cases.
 
-A [chatbot](/agents/chatbot) can be thought of as an agent connected to some messaging interface.
+A [chatbot](/agents/chatbot) can be thought of as an agent connected to a messaging interface, such as Slack.
 
-## How to work with AI agents
+## `CREATE AGENT` Syntax
 
-### Create skills
+Here is the syntax for creating agents.
 
-Start by setting up the skills. Here is how you can create and manage skills using SQL API.
+```sql
+CREATE AGENT my_agent
+USING 
+    skills = ['text_to_sql_skill'],
+    provider = 'openai',
+    model = 'gpt-4',
+    prompt_template = 'Answer the user input in a helpful way using tools',
+    verbose = True,
+    max_tokens = 100;
+```
+
+<Note>
+Users can define a default model in the [MindsDB configuration under `default_llm`](/setup/custom-config) to be used with agents.
```

Comment:
Should we mention this? If users use docker they will need to build new image for this to take effect? Maybe mention that we use this default model

Reply:
@MinuraPunchihewa has implemented the default model such that it must be passed into config.json. I can add info that users need to rebuild Docker image. But otherwise, they cannot use default models if these are not defined.

@MinuraPunchihewa 
Is there any easier way to do it for MindsDB in Docker?

Reply:
@martyna-mindsdb I believe there is going to be a GUI page created for setting the default models? They can also do it by calling the back-end API (PUT /api/config) directly.

Reply:
@MinuraPunchihewa 
Can you please give an example of how users would call this endpoint specifically to add default models? Something like:
```
curl PUT /api/config
...
{ default_model: ...}
```